examples/C/depth/rs-depth.c

```diff
@@ -56,7 +56,7 @@ float get_depth_unit_value(const rs2_device* const dev)
 
         if (1 == is_depth_sensor_found)
         {
-            depth_scale = rs2_get_option(sensor, RS2_OPTION_DEPTH_UNITS, &e);
+            depth_scale = rs2_get_option((const rs2_sensor*)sensor, RS2_OPTION_DEPTH_UNITS, &e);
```

Comment:
Please verify that GCC does not raise a cast warning - it is now passed as rs2_options* param